app/models/normalize_timdex.rb

```diff
@@ -20,10 +20,14 @@ def view_more(q)
   def extract_results(results, norm)
     return unless results['data']['search']['records']
     results['data']['search']['records'].first(5).each do |record|
-      result = Result.new(record['title'], record['sourceLink'])
+      result_title = record['title'] + ' (' + record['identifier'].gsub('MIT:archivespace:','') + ')'
```

Comment:
Take a look at this option and consider if it may be clearer than `gsub` here:
https://ruby-doc.org/core-2.7.0/String.html#method-i-delete_prefix

(ruby is all about syntactic sugar and you'll either really appreciate this or think it's the worst lol)